### PR TITLE
Hotfix/4.4.0 py version fix

### PR DESCRIPTION
**Pull request by maciekswat**
Bumping up versions

### CODE REVIEW

File: conda-recipes/versions.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
-version: 4.4.0
+version: 4.5.0
```
Comment (tjsego) [inline conda-recipes/versions.yaml:1]: Same comment as in main repo. How about 4.4.1 as a patch release?

Comment (maciekswat) [reply conda-recipes/versions.yaml]: Done

File: conda-recipes/versions.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
-version: 4.4.0
+version: 4.4.1
```
Comment (tjsego) [inline conda-recipes/versions.yaml:1]: 4.4.1 or 4.5.0?

**Review by tjsego — CHANGES_REQUESTED** [review]
Minor cleanup here. Otherwise, looks great.

**Review by tjsego — CHANGES_REQUESTED** [review]
Looks good. Just needs previously raised issues resolved.

**Review by tjsego — APPROVED** [review]

**Pull request merged**